vita3k/shader/src/usse_translator_entry.cpp

```diff
@@ -35,7 +35,7 @@ using USSEMatcher = shader::decoder::Matcher<Visitor, uint64_t>;
 
 template <typename V>
 static std::optional<const USSEMatcher<V>> DecodeUSSE(uint64_t instruction) {
-    static const std::vector<USSEMatcher<V>> table = {
+    static const std::array<USSEMatcher<V>, 33> table = {
```

Comment:
you would need to update the matching line in tools/usse-decoder-gen/autogen.py

Reply:
done